### DISCUSSION TITLE
Add custom (re-usable) playlists

### DISCUSSION THREAD

**Discussion by AlphaGameDeveloper** [codetheweb/muse#953] [Ideas]
Maybe add playlists, like
/playlist (or /pl)
/pl create (name:string)
/pl add (name:string)
/pl delete (name:string)
and /pl use (name:string)
etc.
and add the function to chop in chapters with large videos (like [this](https://www.youtube.com/watch?v=LV7Y8jUa6wE)

**Comment by codetheweb** [comment]
does the favorites feature work for this?
